.github/workflows/ci.yml

```diff
@@ -39,14 +39,24 @@ jobs:
           lake -R exe cache get # download cache of mathlib docs.
           lake -R build AliveScaling
 
-      - name: Compile Executable 🧐
+      - name: Compile `mlirnatural` Executable 🧐
```

Comment:
Needs a more descriptive name than MLIR-natural.

Reply:
That's an unrelated change (renaming `mlirnatural`), since the executable already has this name and has had it for a while. This is change is just disambiguating the two executables in CI

Reply:
@tobiasgrosser I'm going to resolve this, since coming up with good names for these tools is beyond the scope of this PR.

Reply:
I guess this is a strict improvement.